userguide/services.rst

```diff
@@ -4,6 +4,7 @@
 Services
 ========
 
+
 The Services section of the GUI is where various services that ship
```

Comment:
"Services that ship with %brand% can be configured, started, or stopped with :guilabel:`Services`.